app/app/v1alpha/app.proto

```diff
@@ -0,0 +1,314 @@
+syntax = "proto3";
+
+package app.app.v1alpha;
+
+import "common/healthcheck/v1beta/healthcheck.proto";
+// Google API
+import "google/api/field_behavior.proto";
+// Protocol Buffers Well-Known Types
+import "google/protobuf/timestamp.proto";
+
+// LivenessRequest represents a request to check a service liveness status
+message LivenessRequest {
+  // HealthCheckRequest message
+  optional common.healthcheck.v1beta.HealthCheckRequest health_check_request = 1 [(google.api.field_behavior) = OPTIONAL];
+}
+
+// LivenessResponse represents a response for a service liveness status
+message LivenessResponse {
+  // HealthCheckResponse message
+  common.healthcheck.v1beta.HealthCheckResponse health_check_response = 1;
+}
+
+// ReadinessRequest represents a request to check a service readiness status
+message ReadinessRequest {
+  // HealthCheckRequest message
+  optional common.healthcheck.v1beta.HealthCheckRequest health_check_request = 1 [(google.api.field_behavior) = OPTIONAL];
+}
+
+// ReadinessResponse represents a response for a service readiness status
+message ReadinessResponse {
+  // HealthCheckResponse message
+  common.healthcheck.v1beta.HealthCheckResponse health_check_response = 1;
+}
+
+/*
+
+   This API is under development and, therefore, some of its entities and
+   endpoints are not implemented yet. This section aims to give context about the
+   current interface and how it fits in the App vision.
+
+   # App
+
+   The App domain is responsible of storing data that will later be used for
+   processing unstructured data. App will support the following types of
+   data:
+
```

Comment:
These content is wrong copied?

Reply:
right. 🤣  just removed. help to review again. 🙇‍♂️